Index: src/components/menu.tsx
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>import React from 'react';\r\nimport Switch from \"./switch\";\r\nimport {useSelector} from \"react-redux\";\r\n\r\n\r\n\r\ninterface MenuProps {\r\n    isOpen: boolean;\r\n    lightTheme: boolean\r\n}\r\n\r\nconst Menu: React.FC<MenuProps> = ({isOpen,lightTheme}) => {\r\n    const photoUrl = useSelector((state:any) => state.photo.photoUrl)\r\n    return (\r\n        <div className={`menu ${isOpen ? 'open' : ''}`}>\r\n\r\n            <img className={'avatar_manu'} src={photoUrl} alt={`ph`}/>\r\n\r\n            <Switch/>\r\n        </div>\r\n    );\r\n};\r\n\r\nexport default Menu;
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/src/components/menu.tsx b/src/components/menu.tsx
--- a/src/components/menu.tsx	
+++ b/src/components/menu.tsx	
@@ -4,6 +4,7 @@
 
 
 
+
 interface MenuProps {
     isOpen: boolean;
     lightTheme: boolean
